fiftyone/factory/repos/delegated_operation_doc.py

```diff
@@ -46,32 +46,36 @@ def __init__(
         self.pinned = False
         self.completed_at = None
         self.failed_at = None
+        self.scheduled_at = None
         self.result = None
         self.id = None
         self._doc = None
         self.metadata = None
 
     def from_pymongo(self, doc: dict):
         # required fields
-        self.operator = doc["operator"]
-        self.queued_at = doc["queued_at"]
-        self.run_state = doc["run_state"]
-        self.label = doc["label"] if "label" in doc else None
-        self.updated_at = doc["updated_at"] if "updated_at" in doc else None
+        self.operator = doc.get("operator")
+        self.queued_at = doc.get("queued_at")
+        self.run_state = doc.get("run_state")
 
         # optional fields
-        self.delegation_target = (
-            doc["delegation_target"] if "delegation_target" in doc else None
-        )
-        self.started_at = doc["started_at"] if "started_at" in doc else None
-        self.completed_at = (
-            doc["completed_at"] if "completed_at" in doc else None
-        )
-        self.failed_at = doc["failed_at"] if "failed_at" in doc else None
-        self.pinned = doc["pinned"] if "pinned" in doc else None
-        self.dataset_id = doc["dataset_id"] if "dataset_id" in doc else None
-        self.run_link = doc["run_link"] if "run_link" in doc else None
+        self.delegation_target = doc.get("delegation_target", None)
```

Comment:
technically `None` is already the default so not necessary, but it's fine to leave it